src/docs.md

```diff
@@ -230,6 +230,21 @@ cadenzaClient.on('selectObjects:ok', (event) => {
 });
 ```
 
+### Change visibility of Layers in a Workbook Map
+
+<small>API: [CadenzaClient#setLayerVisibility](./classes/CadenzaClient.html#setLayerVisibility)</small>
+
+To change the visibility of a layer in an already shown map view, pass the layer path and the desired visibility.
+The operation is asynchronous and completion is signaled with an event. 
+
+```javascript
+cadenzaClient.on('setLayerVisibility:success', (event) => {
+  console.log('Layer visibility change is complete!');
+});
```

Comment:
If you return the Promise, there's no need for clients to listen to that event.

Reply:
Indeed.